scripts/analysis.py

```diff
@@ -916,26 +887,23 @@ def multi_line_plot(dictionary, timestep,
     Parameters
     ----------
     dictionary: dictionary
-        dictionary with list of timestep progressions
-    timestep: int
-        timestep of simulation (linspace)
-    xlabel: string
+        dictionary with value: list of timestep progressions
```

Comment:
The keys are the names that describe the timesteps.
I've added them to the documentation.